packages/shared/src/lib/featureManagement.ts

```diff
@@ -16,11 +16,8 @@ export class Feature<T extends JSONValue> {
 const feature = {
   feedVersion: new Feature('feed_version', 15),
   lowImps: new Feature('feed_low_imps'),
-  bookmarkOnCard: new Feature('bookmark_on_card', false),
-  readingStreaks: new Feature(
-    'reading_streaks',
-    ReadingStreaksExperiment.Control,
-  ),
+  bookmarkOnCard: new Feature('bookmark_on_card', true),
+  readingStreaks: new Feature('reading_streaks', ReadingStreaksExperiment.V2),
```

Comment:
Let's not change this from control, we need it for existing streaks user experiment

Reply:
Ah, my bad. Was testing. Accidentally pushed.